packages/x-grid-data-generator/src/renderer/renderStatus.tsx

```diff
@@ -58,6 +58,7 @@ const Status = React.memo((props: StatusProps) => {
   }
   return (
     <Chip
+      // @ts-ignore
```

Comment:
it seems you just put a ts-ignore at every compile error instead of fixing the issue